src/Hng.Web/Bootstrappers/ServicesBootstrapper.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Hng.Application.Interfaces;
+using Hng.Application.Services;
+using Hng.Domain.Models;
+using Hng.Infrastructure.Repository.Interface;
+using Hng.Infrastructure.Repository.Mock;
+using Hng.Infrastructure.Repository.SqlData;
+using Hng.Web.Mappers;
+
+namespace Hng.Web.Bootstrappers
+{
+    public static class ServicesBootstrapper
```

Comment:
What in christ's name is this?

Reply:
the term "bootstrapper" refers to a process or a set of initial steps necessary to start a larger and more complex process. Registration of repositories and services is one of those initial steps. The reason this file exists is to prevent having to register repositories and core services in different locations since the web needs it and also the unit test application

Team members only need to add their services to this file once and it is available in any part of the application where registration is required (test and web project)

This file is just a dedicated place for registering repositories and custom services. The naming might not be right and you can suggest a name and I'll change it to that